tutorials/pubsub-spring-dedup-messages/pubsub-spring/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.example</groupId>
+  <artifactId>pubsub-spring</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-parent</artifactId>
+    <version>2.2.4.RELEASE</version>
+  </parent>
+
+  <properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+  </properties>
+  <dependencies>
+
+
+    <dependency>
+      <groupId>org.springframework.cloud</groupId>
+      <artifactId>spring-cloud-gcp-pubsub-stream-binder</artifactId>
+      <version>1.2.0.RELEASE</version>
```

Comment:
I've heard it's good practice to keep [versions as variables](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/88ae90245aa8a6f6ddc84f8fdc47ada35326aeb3/dataflow/encryption-keys/pom.xml#L33-L37) in the `<properties>` section.

Reply:
This `pom.xml` is really short. I would like to keep it simple like it is. :)